torch/csrc/jit/operator_upgraders/utils.h

```diff
@@ -4,6 +4,7 @@
 #include <torch/csrc/jit/operator_upgraders/version_map.h>
 #include <string>
 #include <vector>
+#include <stdint.h>
```

Comment:
should probably be
```suggestion
#include <cstdint>
```
right?